src/main/java/org/deepsymmetry/beatlink/Util.java

```diff
@@ -959,6 +958,29 @@ public static String phraseLabel(final SongStructureEntry phrase) {
         }
     }
 
+    /**
+     * Scan a network interface to find if it has an address space which matches the device we are trying to reach.
+     * If so, return the address specification.
+     *
+     * @param aDevice the DJ Link device we are trying to communicate with
+     * @param networkInterface the network interface we are testing
+     * @return the address which can be used to communicate with the device on the interface, or null
+     */
+    public static InterfaceAddress findMatchingAddress(DeviceAnnouncement aDevice, NetworkInterface networkInterface) {
```

Comment:
I don’t use argument names like `aDevice`, I would use `announcement` or `device` here.

Reply:
(And I understand this came from my old code, so you don’t have to change it, but I’d like to try to remember to do so in my final cleanup phase. 😄)

Reply:
Always happy to adjust things, sooner the better. Ill do it now.